lib/builtins/deploy-delegates/lambda-deployer/helper.js

```diff
@@ -232,13 +248,48 @@ function _addEventPermissions(lambdaClient, skillId, functionArn, profile, callb
     });
 }
 
+function _getLambdaCustomValue(alexaRegion, userConfig, parameter) {
+    return R.path(['regionalOverrides', alexaRegion, 'lambda', parameter], userConfig)
+    || R.path(['lambda', parameter], userConfig);
+}
+
+function _getLambdaFunctionName(alexaRegion, userConfig, skillName, profile) {
```

Comment:
nit: Would be nice to have a quick jsdoc comments above these new methods that describe it, just something like:

```
/*
 * Loads the function name from config or generates a unique function name
 *
 */
```